Server/uploadpackage.R

```diff
@@ -27,7 +27,40 @@ uploaded_pkgs <- reactive({
   
   names(uploaded_pkgs) <- tolower(names(uploaded_pkgs))
   uploaded_pkgs$package <- trimws(uploaded_pkgs$package)
-  uploaded_pkgs$version <- trimws(uploaded_pkgs$version)
+  
+  # https://stackoverflow.com/questions/33589591/check-if-package-name-belongs-to-a-cran-archived-package
+  get_archived <- function(cran = getOption("repos")){
+    if (is.null(cran)) cran <- "http://cran.rstudio.com/"
+    con <- gzcon(url(sprintf("%s/%s", cran, "src/contrib/Meta/archive.rds"), open = "rb"))
```

Comment:
I keep getting this error:

![image](https://user-images.githubusercontent.com/17878953/156225735-0b75bc68-83ab-4bbb-a531-f2d250f2b0c5.png)


I think it's because my RStudio profile is pointing to more than one repo besides CRAN:

![image](https://user-images.githubusercontent.com/17878953/156225420-9e2c974c-5de2-4a50-832e-08f14cfce6e0.png)

If you want CRAN only, you should request it explicitly by using: 
`cran = getOption("repos")[names(getOption("repos"))== "CRAN"]`

Reply:
Doing this differently, won't need `getOption("repos")` anymore

Reply:
I have updated this.
Now using this .csv
![image](https://user-images.githubusercontent.com/75215958/156377019-defedbeb-ff17-4dfd-8b5c-3029d1abac1c.png)

NOTE: package LaOveja was flagged by riskmetric as 'pkg_missing' and will be removed. Did you misspell it? 
NOTE: package stanly was not found in the CRAN archive and will be removed.

![image](https://user-images.githubusercontent.com/75215958/156377277-98dd7c7e-5497-4dea-afaf-58e31c136ec0.png)

Reply:
However, when you select on this particular package, you get
```{r, eval=FALSE}
No rows were returned from query
SELECT * 
FROM community_usage_metrics
WHERE id = 'samplesizeCMH'
Warning in min(year) : no non-missing arguments to min; returning Inf
Warning in min(month) : no non-missing arguments to min; returning Inf
Warning: Error in if: argument is of length zero
  1: runApp
Warning in min(downloads) :
  no non-missing arguments to min; returning Inf
Warning in max(downloads) :
  no non-missing arguments to max; returning -Inf
```